app/src/main/java/com/example/airbnb/presentation/where/WhereViewModel.kt

```diff
@@ -0,0 +1,38 @@
+package com.example.airbnb.presentation.where
+
+import android.util.Log
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.example.airbnb.core.view.UiState
+import com.example.airbnb.data.ServicePool
+import kotlinx.coroutines.launch
+
+class WhereViewModel : ViewModel() {
+    private val _whereLiveData: MutableLiveData<UiState<List<WhereItem>>> = MutableLiveData()
+    val whereLiveData: MutableLiveData<UiState<List<WhereItem>>> = _whereLiveData
+
+    init {
+        getWhere()
+    }
+
+    private fun getWhere() = viewModelScope.launch {
+        _whereLiveData.value = UiState.Loading
+        runCatching {
+            ServicePool.whereService.getWhere()
+        }.onSuccess {
+            val whereList = it.data ?: emptyList()
+            val whereItems: List<WhereItem> = whereList.map { whereList ->
+                WhereItem(
+                    locationName = whereList.regionName,
+                    locationImageUrl = whereList.imageUrl
+                )
+            }
+
+            _whereLiveData.value = UiState.Success(whereItems)
+            Log.d("wv", "성공")
+        }.onFailure {
+            Log.d("wv", it.message.toString())
```

Comment:
```suggestion
            _whereLiveData.value = UiState.Failure(it.message.toString())
```
로그는 테스트 후에 지워주세요~~ 그리고 onFailure 일 때도 livedata에 값을 넣어주면 실패했을 때 에러메시지를 얻는 분기 처리도 가능합니다!